formats/detailed_guide/publisher/details.json

```diff
@@ -0,0 +1,39 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "type": "object",
+  "additionalProperties": false,
+  "required": [
+    "body",
+    "first_published_at"
+  ],
+  "properties": {
+    "body": {
+      "type": "string"
+    },
+    "first_published_at": {
+      "type": "string",
+      "format": "date-time"
+    },
+    "image": {
+      "$ref": "#/definitions/image"
+    },
+    "scheduled_publication": {
```

Comment:
Probably `DetailedGuide.first.scheduled_publication` but it shouldn't be public, so removing.

Postponed doing this until I did the backend work just to be sure of it.